examples/simple-platform/platform-structure.md

```diff
@@ -0,0 +1,45 @@
+# Platform structure
+
+Our platform has two server nodes. One runs the `platform-deploy` hypervisor 
+and the other runs the `web-auth-endpoint` hypervisor. We also have two 
+browser nodes - one for the accounts app and another for the template app. 
+These nodes run the React apps in the browser.
+
+The `platform-deploy` node hosts the accounts and template chains, acting as 
+the server for the two apps. It is the blockmaster and thus is responsible for 
```

Comment:
I think we should shy away from the term server. This is not actually a server for the apps or a server at all - it is an Interbit node. The apps are served from their own static servers elsewhere in our production environment, and in dev mode they are served from their own webpack servers.

Reply:
I think the key concept is the difference in node lifecycles. The browser nodes are transient, but the back-end nodes are long running, keeping the chains alive for as long as they run (as near as we currently get to chain persistence).